src/NLog.Targets.Syslog/MessageSend/Udp.cs

```diff
@@ -11,19 +12,21 @@ namespace NLog.Targets.Syslog.MessageSend
     internal class Udp : MessageTransmitter
     {
         private readonly UdpClient udp;
+        private readonly IPEndPoint endPoint;
         private volatile bool disposed;
 
         public Udp(UdpConfig udpConfig) : base(udpConfig.Server, udpConfig.Port)
         {
-            udp = new UdpClient(IpAddress, Port);
+            endPoint = new IPEndPoint(System.Net.IPAddress.Parse(IpAddress), Port);
+            udp = new UdpClient();
         }
 
         public override Task SendMessageAsync(ByteArray message, CancellationToken token)
         {
             if (token.IsCancellationRequested)
                 return Task.FromResult<object>(null);
 
-            return udp.SendAsync(message, message.Length);
+            return udp.SendAsync(message, message.Length, endPoint);
```

Comment:
[It seems](https://docs.microsoft.com/en-us/dotnet/api/system.net.sockets.udpclient.-ctor?view=netstandard-1.3) that the old code constructor is supported